modules/tinymce/src/plugins/searchreplace/test/ts/browser/SearchReplaceDialogTest.ts

```diff
@@ -112,4 +114,14 @@ describe('browser.tinymce.plugins.searchreplace.SearchReplaceDialogTest', () =>
     findAndAssertFound(editor, 3);
     TinyUiActions.closeDialog(editor);
   });
+
+  it('TINY-9148: Scroll should remain in place after the dialog is closed', async () => {
```

Comment:
Are we sure that this test works? Does it fail if you remove your fix. Since the fiddle scrolls the parent window pane not the iframe contents and the sink is in the parent document not the iframe document. So I would expect that the editor container would need say margin of 2000px or something to produce a scroll.

- [ ] Confirm that this test works if you remove the fix

Reply:
You're right. I was testing with the `autoresize` plugin on. Thanks for pointing it out!